content/03.aim1.md

```diff
@@ -29,4 +29,34 @@ We will evaluate resulting models for their applicability to rheumatic disease a
 
 ### Results
 
-TBD.
+#### VAE test on simulated single cell datasets
+#### 1. Simulation data generation: 
+Simulated single cell data was generated by splatter [@doi:10.1186/s13059-017-1305-0]. 
+
+Parameters used:
+* nCells - The number of cells to simulate: 500 - 5000
+* nGenes - The number of genes to simulate: 20000 - 6000
+* nGroups - The number of celltypes: 5 - 15
+* outlier - probability of a gene that is an expression outlier:  0.1 - 0.5
+* default parameter is ncells = 600, nGenes = 20000, celltypes = 5, batchsize = 1
+
+Simulation statistics under different parameters:
+
+![](images/mean.gene.exp.bmp)
```

Comment:
I am confused about what this figure is showing? Mean expression on Y?

Reply:
It is the mean expression of genes from different samples for different simulation parameters. I will describe it clearly